src/AdoPat/TokensHttpClientWrapper.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Authentication.AdoPat
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.VisualStudio.Services.DelegatedAuthorization;
+    using Microsoft.VisualStudio.Services.DelegatedAuthorization.Client;
+    using Microsoft.VisualStudio.Services.WebApi;
+
+    /// <summary>
+    /// A thin wrapper around a <see cref="TokensHttpClient"/> that implements the <see cref="ITokensHttpClientWrapper"/> interface.

```

Comment:
Is this class `TokensHttpClient` implemented in a different PR?

Reply:
oh! I see it's a VS library.

Reply:
Yup. This is a class in another library. 😄